packages/@tinacms/schema-tools/src/validate/schema.ts

```diff
@@ -16,7 +16,7 @@ import { name } from './properties'
 import { findDuplicates } from '../util'
 import { TinaFieldZod } from './fields'
 import { tinaConfigZod } from './tinaCloudSchemaConfig'
-const FORMATS = ['json', 'md', 'markdown', 'mdx'] as const
+const FORMATS = ['json', 'md', 'markdown', 'mdx', 'toml', 'yaml'] as const
```

Comment:
@logan-anderson does it make sense to have all of these formats together? The existing ones apply to the format property while the latter apply to the frontmatterFormat property

Reply:
@kldavis4 I was able to add support for those formats as well. I will update the changeset and the description.

But doing { format: 'toml' } will now allow you to have toml files.

Reply:
Ok, thanks for the clarification 👍🏻